src/main/java/uk/gov/hmcts/ccd/domain/service/listevents/ListEventsOperation.java

```diff
@@ -10,4 +10,5 @@
 
     List<AuditEvent> execute(String jurisdiction, String caseTypeId, String caseReference);
 
+    AuditEvent execute(String jurisdiction, String caseTypeId, Long eventId);
```

Comment:
wonder whether having multiple execute methods called simply 'execute' is starting to be a bit confusing.
Not sure but, shall we start giving more explanatory names? 
Like getEvents and getEvent or similar?

Reply:
Yes again its a separate use case to get event by id and should not be part of ListEventOperation. Will try to separate that out too.

Reply:
I haven't created a new implementation for GetEventOperation, rather renamed ListEventsOperation to GetEventsOperation. Currently I see some inconsistency with multiple execute methods in other areas. Will leave out abstraction and renaming of this operation as there is a scope for re-usability of authorisation and security classification code throughout the app. Probably its a good idea to get all Devs to agree on a common architecture of the application code.